src/index.ts

```diff
@@ -2,6 +2,11 @@ import { parseOptions } from './Options';
 import { runWithServer } from './Server';
 import { runSuite } from './Suite';
 
+export type AxeParams = {
```

Comment:
Ah, so we export the type from here, and then in an app using this (e.g. Learning Platform) we'd do something like

```ts
import type {AxeParams} from '@chanzuckerberg/axe-storybook-testing';

declare module '@storybook/react' {
  interface Parameters {
    axe?: AxeParams;
  }
}
```

I hadn't thought of doing that, but it makes a lot of sense 👍.

This does mean we'll be shipping a bunch of type definitions that no one will ever use (e.g. for all the files making up this project). Feels a bit unecessary.

One alternative would be to provide an `index.d.ts` at the root of this project that defines these params.

Pro: we wouldn't ship unnecessary types/files with the package.
Con: we'd have to duplicate the type in index.d.ts, and keep it up to date.

Not sure what the right answer is. What do you think about the tradeoffs?

Reply:
I think what I'd ideally like is to be able to ship one `.d.ts` file that includes types we want to expose. So that way we don't ship a bunch of types people won't use & the ones we do want people to use get updated automatically.

Maybe what we can do is:
1. set `declarationDir ` to `./types` so our types files get emitted there
2. have a `externalTypes.ts` file that exports things like `AxeParams`
3. add `types/externalTypes.d.ts` to `files` in `package.json`

I think theoretically that just ships the one file with types we want?